### PR TITLE
Visibility: an update after component's unmount throws exception

### LINKED ISSUE #2570: Visibility: Cannot read property 'getBoundingClientRect' of null
✔ BUGS → This form is required:

### Steps
I could not track it down.
It is available on your site:
https://react.semantic-ui.com/layouts/homepage

### Expected Result
Responsiveness
### Actual Result
Crashing between 768px and 992px width and less than 320 px.
### Version

```js
Uncaught TypeError: Cannot read property 'getBoundingClientRect' of null
    at Visibility.computeCalculations (Visibility.js:238)
    at Visibility._this.update (Visibility.js:72)
computeCalculations @ Visibility.js:238
Visibility._this.update @ Visibility.js:72
requestAnimationFrame (async)
Visibility._this.handleUpdate @ Visibility.js:67
(anonymous) @ EventTarget.js:31
arrayEach @ _arrayEach.js:15
forEach @ forEach.js:38
(anonymous) @ EventTarget.js:30
(anonymous) @ _createBaseFor.js:17
baseForOwn @ _baseForOwn.js:13
(anonymous) @ _createBaseEach.js:17
forEach @ forEach.js:38
(anonymous) @ EventTarget.js:24
```

### PR DESCRIPTION
**Pull request by VikVelev**
The responsiveness on your doc page was broken and wasn't considering devices other than mobile and desktop.

Fixes #2570

### CODE REVIEW

**Comment by layershifter** [comment]
To team: don't merge this, the problem is in suddenly unmounted Visibility component and should be fixed there.

File: docs/app/Layouts/HomepageLayout.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ class DesktopContainer extends Component {
     const { fixed } = this.state
 
     return (
-      <Responsive {...Responsive.onlyComputer}>
+      <Responsive {...Responsive.onlyComputer} minWidth={768}>
```
Comment (levithomason) [inline docs/app/Layouts/HomepageLayout.js:72]: I'm not sure on this.  The props being spread in `Responsive.onlyComputer` include a minWidth of 992.  Why is is that we need to override this to 768?

https://github.com/Semantic-Org/Semantic-UI-React/blob/8ccee0f75fb2a9802bbaf33394182dc97e8b5d2e/src/addons/Responsive/Responsive.js#L56

If 768 is indeed correct, then we should not be spreading the other value and then overridding it.

File: docs/app/Layouts/HomepageLayout.js [hunk 2]
```diff
@@ -114,7 +114,7 @@ class MobileContainer extends Component {
     const { sidebarOpened } = this.state
 
     return (
-      <Responsive {...Responsive.onlyMobile}>
+      <Responsive {...Responsive.onlyMobile} minWidth={0}>
```
Comment (levithomason) [inline docs/app/Layouts/HomepageLayout.js:117]: This one also is specifically spreading a different value for minWidth.  Why is it that we need to override this to 0?

https://github.com/Semantic-Org/Semantic-UI-React/blob/8ccee0f75fb2a9802bbaf33394182dc97e8b5d2e/src/addons/Responsive/Responsive.js#L54

As above, if minWidth 0 is correct then we should not be spreading `onlyMobile` here, or, we need to update the values of `onlyMobile`.

**Review by levithomason — COMMENTED** [review]
@layershifter Can you take a look here?

**Comment by layershifter** [comment]
See #2635 for details.

**Pull request closed without merging**